x/session/keeper/session_hydrator.go

```diff
@@ -67,22 +66,22 @@ func (k Keeper) HydrateSession(ctx context.Context, sh *sessionHydrator) (*types
 	logger := k.Logger().With("method", "hydrateSession")
 
 	if err := k.hydrateSessionMetadata(ctx, sh); err != nil {
-		return nil, types.ErrSessionHydration.Wrapf("failed to hydrate the session metadata: %v", err)
```

Comment:
Did you removed these because they are unnecessary wrappers around the error?

Reply:
It was causing the error message to include the path to the errors.go where the inner error is defined, making the message unnecessarily verbose.